src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/ComputedTilesToComputedTrends.scala

```diff
@@ -0,0 +1,45 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.{ComputedTile, ComputedTrend}
+import org.apache.commons.math3.stat.regression.SimpleRegression
+import org.apache.spark.rdd.RDD
+
+object ComputedTilesToComputedTrends {
+
+  def apply(computedTiles: RDD[ComputedTile]): RDD[ComputedTrend] = {
+    val groupedTiles = computedTiles.groupBy(tileGroup)
+    groupedTiles.flatMap[ComputedTrend]((group) => {
+      val tiles = group._2.toSeq
+
+      val regressionResult = linearRegression(tiles)
```

Comment:
Just as a general point: using linear regression for anomaly detection is very naive, to the point of not being useful in many real-world situations. From my understanding, there are three types of timeseries anomalies that we normally care about: spikes, step changes and shifts. With a standard linear model, we can at best hope to detect spikes.

A still simple but slightly less naive model would incorporate at least a trend term (e.g. by rolling median for the last N timeframes) and a seasonality term (e.g. via fast-fourier on the last N timeframes). There is a good [article by AirBnB](https://medium.com/airbnb-engineering/anomaly-detection-for-airbnb-s-payment-platform-e3b0ec513199) on a simple model like this.